<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>PolygonPreview</class>
    <widget class="QWidget" name="PolygonPreview">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>576</width>
                <height>425</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Advanced</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="4">
                <layout class="QGridLayout" name="mainLayout">
                    <item row="2" column="1">
                        <widget class="QSpinBox" name="spinBox">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>71</width>
                                    <height>27</height>
                                </size>
                            </property>
                            <property name="minimum">
                                <number>1</number>
                            </property>
                            <property name="maximum">
                                <number>9</number>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="0">
                        <widget class="QLabel" name="points">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>points:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="3" column="1">
                        <widget class="QLabel" name="Y">
                            <property name="minimumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>Y:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="3" column="0">
                        <widget class="QLabel" name="X">
                            <property name="minimumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>X:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="rotation">
                            <property name="text">
                                <string>rotation:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QTextEdit" name="editRotation">
                            <property name="maximumSize">
                                <size>
                                    <width>71</width>
                                    <height>27</height>
                                </size>
                            </property>
                            <property name="placeholderText">
                                <string>0</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="2" column="4">
                <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>150</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="1" column="0">
                <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>40</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="1" column="1">
                <spacer name="horizontalSpacer">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>40</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="4" column="4">
                <widget class="QPushButton" name="buttonPresets">
                    <property name="text">
                        <string>presets</string>
                    </property>
                </widget>
            </item>
            <item row="3" column="4">
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>10</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="1" column="3">
                <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>40</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="1" column="2">
                <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>40</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
